.github/workflows/build_steps.yml

```diff
@@ -87,12 +87,16 @@ jobs:
           disk-root: "D:"  # This is also the checkout directory. Total size 12GB.
         continue-on-error: true
 
+      - name: Install VS2022 BuildTools 17.9.7
+        if: matrix.os == 'windows'
+        run: choco install -y visualstudio2022buildtools --version=117.9.7.0 --params "--add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 --installChannelUri https://aka.ms/vs/17/release/180911598_-255012421/channel"
+
       - name: Enable Windows compiler commands
         if: matrix.os == 'windows'
         uses: ilammy/msvc-dev-cmd@v1.12.1
         with:
```

Comment:
I don't think we need the with clause anymore. It was added a while ago because `14.40` broke the build but `14.39` was still around. Can you check if it compiles without it and if so push it without this.

Reply:
Do you mean removing the `toolset: 14.40` tag?